workflows/chipseq.nf

```diff
@@ -508,14 +510,18 @@ workflow CHIPSEQ {
                 .map { meta, saf -> [ meta.id, meta, saf ] }
                 .set { ch_ip_saf }
 
+            // ch_ip_saf.dump()
+
             ch_ip_control_bam
                 .map { meta, ip_bam, control_bam -> [ meta.antibody, meta, ip_bam ] }
                 .groupTuple()
                 .map { it -> [ it[0], it[1][0], it[2].flatten().sort() ] }
+                // .dump()
```

Comment:
```suggestion
```

Reply:
Fixed in [277bd11](https://github.com/nf-core/chipseq/pull/279/commits/277bd119f4747960d0059df30f7a3cea8fccb625)